tests/c-sharp.test

```diff
@@ -122,7 +122,7 @@
 60015  cs/UNI-14131.cfg                             cs/UNI-14131.cs
 60016  cs/UNI-11662.cfg                             cs/UNI-11662.cs
 60018  common/UNI-18777.cfg                         cs/UNI-18777.cs
-60019  common/empty.cfg                             cs/UNI-18780.cs
+##60019  common/empty.cfg                             cs/UNI-18780.cs
```

Comment:
@guy-maurel What is the reason to disable this test? If the code changes make the test fail, we should either adjust the code or the test, depending on what is applicable

Reply:
Yes, look at #4282